packages/teams-js/src/public/runtime.ts

```diff
@@ -206,8 +211,15 @@ export const versionConstants: Record<string, Array<ICapabilityReqs>> = {
  * @returns runtime which describes the APIs supported by the legacy host client.
  */
 export function generateBackCompatRuntimeConfig(highestSupportedVersion: string): IRuntime {
+  generateBackCompatRuntimeConfigLogger('generating back compat runtime config for %s', highestSupportedVersion);
+
   let newSupports = { ...teamsRuntimeConfig.supports };
 
+  generateBackCompatRuntimeConfigLogger(
+    'Supported capabilities in config before updating based on highestSupportedVersion: %o',
+    JSON.stringify(newSupports),
+  );
```

Comment:
Out of curiosity, do you know what this output looks like when run from minimized code? Is it something we will still be able to interpret?

Reply:
It appears to be stable in minified code based off what I've seen from the bug I'm looking at. %i and %o both seem to work as expected